cmd/kubeconfig.go

```diff
@@ -47,27 +49,27 @@ users:
     client-key-data: {{.ClientKey}}
 `))
 
-	// userCmd creates new certs and kubeConfig for a user
-	userCmd = &cobra.Command{
-		Use:   "user [command]",
+	// kubeconfigCmd creates new certs and kubeConfig for a user
+	kubeconfigCmd = &cobra.Command{
+		Use:   "kubeconfig [command]",
 		Short: "Manage user access",
```

Comment:
change to a more meaningful description, now that we are no longer using the term `user`

Reply:
Maybe "Dump kubeconfig for desired user".
I'm having difficulty phrasing it correctly.

Reply:
maybe instead of "dump" use: `Print kubeconfig for desired user to stdout`